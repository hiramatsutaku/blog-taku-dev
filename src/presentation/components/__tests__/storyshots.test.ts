import initStoryshots from '@storybook/addon-storyshots';
import { styleSheetSerializer } from 'jest-styled-components/serializer';

expect.addSnapshotSerializer(styleSheetSerializer);

initStoryshots({});
