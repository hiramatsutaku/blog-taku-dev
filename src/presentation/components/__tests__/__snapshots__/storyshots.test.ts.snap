// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Footer Default Story 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-top: 80px;
}

<footer
  className="c0"
>
  <small>
    2020©hirataku
  </small>
</footer>
`;

exports[`Storyshots Header Default Story 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  list-style: none;
  margin-top: 0;
}

.c2 {
  margin-left: 1em;
}

<header
  className="c0"
>
  <a
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    hirataku.dev
  </a>
  <nav>
    <ul
      className="c1"
    >
      <li
        className="c2"
      >
        <a
          href="/"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          Home
        </a>
      </li>
      <li
        className="c2"
      >
        <a
          href="/about"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          About
        </a>
      </li>
    </ul>
  </nav>
</header>
`;

exports[`Storyshots Post Default Story 1`] = `
.c0 {
  padding: 0;
}

<div
  className="c0"
/>
`;

exports[`Storyshots Title Default Story 1`] = `
.c0 {
  font-weight: 'bold';
}

<h1
  className="c0"
>
  タイトル
</h1>
`;
